docs/docs/integrations/chat/ollama_functions.ipynb

```diff
@@ -16,6 +16,11 @@
    "source": [
     "# OllamaFunctions\n",
     "\n",
+    ":::{.callout-caution}\n",
+    "\n",
+    "This class is being deprecated. Please use [ChatOllama](./ollama.ipynb) instead.\n",
```

Comment:
My bad, did not see there's already a warning below it. Probably OK to keep just the existing warning.